tests/pattern-tests.js

```diff
@@ -205,6 +215,34 @@ function testPatterns(Prism) {
 		}
 	}
 
+	/**
+	 * Returns whether the given node either is or is a child of what is effectively a Kleene star.
+	 *
+	 * @param {import("regexpp/ast").Node} node
+	 * @returns {boolean}
+	 */
+	function underAStar(node) {
+		if (node.type === "Quantifier" && node.max > 10) {
+			return true;
+		} else if (node.parent) {
+			return underAStar(node.parent);
+		} else {
+			return false;
+		}
+	}
+
+	/**
+	 * @param {Iterable<T>} iter
+	 * @returns {T | undefined}
+	 * @template T
+	 */
+	function firstOf(iter) {
+		for (const item of iter) {
+			return item;
+		}
+		return undefined;
+	}
```

Comment:
This function can be replaced with destructuring (`[first] = iter`).

Reply:
Good point.